diff --git i/src/man.c w/src/man.c
index 8b09db1..88d42ff 100644
--- i/src/man.c
+++ w/src/man.c
@@ -4461,9 +4461,26 @@ int main (int argc, char *argv[])
 					if (!section && maybe_section &&
 					    CTYPE (isdigit, nextarg[0]))
 						gripe_no_name (nextarg);
-					else
+					else {
+#if __MVS__
+            char* fmt = "/bin/man %s %s 2>/dev/null";
+            char* cmd = malloc(strlen(nextarg)+strlen(section)+strlen(fmt)+1);
+            int rc = 0;
+            if (cmd) {
+              sprintf(cmd, fmt, section, nextarg);
+              rc = system(cmd);
+            }
+            if (!cmd || rc) {
+					    gripe_no_man (nextarg, section);
+            }
+            if (cmd) {
+              free(cmd);
+            }
+          }
+#else
 						gripe_no_man (nextarg,
 						              section);
+#endif
 				}
 			}
 		} else {
